content/post/119-how-we-built-this.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2021-01-21
+url: /2021-01-21-how-we-built-this/
+title: Integrating IPFS and Brave – How We Built This
```

Comment:
This is just a proposal – feel free to tweak.
It was team effort, no need to be so serious :)

```suggestion
title: How We Put IPFS in Brave
```